Lib/email/utils.py

```diff
@@ -345,6 +346,8 @@ def localtime(dt=None, isdst=-1):
     to divine whether summer time is in effect for the specified time.
 
     """
+    if isdst != -1:
+        warnings._deprecated("The isdst parameter to localtime", remove=(3, 13))
```

Comment:
Unfortunately, the easy route here makes for an ugly message: `'The isdst argument to localtime' is deprecated ...`.  We need to specify our own message to make it prettier:

```suggestion
        warnings._deprecated(
            "The 'isdst' parameter to 'localtime'",
            message='{name} is deprecated and slated for removal in Python {remove},
            remove=(3, 13),
            )
```